webflow-designer-extension-cli/public/styles.css

```
@@ -199,75 +202,170 @@ body {
+  margin: 0 0 12px;
+  font-size: 13.82px;
+  line-height: 16px;
   color: var(--text-colour--secondary);
 }
 
+.icon {
+  display: inline-block;
+  width: 1em;
+  height: 1em;
+  flex-shrink: 0;
+  background-color: currentColor;
```

Stylelint (17.4.0)

[error] Expected "currentColor" to be "currentcolor" (value-keyword-case)

(value-keyword-case)